<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mp3Player</class>
 <widget class="QMainWindow" name="mp3Player">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mp3Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>631</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QPushButton" name="btnNext">
       <property name="text">
        <string>Next</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnPrev">
       <property name="text">
        <string>Prev</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btnPause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnPlayTrack">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblVolume">
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider">
       <property name="sliderPosition">
        <number>20</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>701</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Tab1">
     <attribute name="title">
      <string>Tracks</string>
     </attribute>
     <widget class="QTableWidget" name="tracksPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Tab2">
     <attribute name="title">
      <string>Now Playing</string>
     </attribute>
     <widget class="QTableWidget" name="playlistPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>701</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnScanDir">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>97</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan..</string>
    </property>
   </widget>
   <widget class="QLabel" name="TrackDuration">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>450</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="TrackTitle">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>431</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="trackPosSlider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
